source/imaer-gml/src/main/java/nl/overheid/aerius/gml/v0_5/source/road/CustomVehicle.java

```diff
@@ -31,11 +31,11 @@
  *
  */
 @XmlRootElement(name = "CustomVehicle", namespace = CalculatorSchema.NAMESPACE)
-@XmlType(name = "CustomVehicleType", namespace = CalculatorSchema.NAMESPACE, propOrder = {"description", "emissions"})
+@XmlType(name = "CustomVehicleType", namespace = CalculatorSchema.NAMESPACE, propOrder = {"description", "emissionFactors"})
```

Comment:
Proporder looks at the name of the property (getter/setter), and as I updated the (overriden) getter from `getEmissions` to `getEmissionFactors`, I also had to change something in this proporder. It will still expect `emission` as the element name for these old versions, that is defined in the getter annotation (and that should be covered by roundtrip tests).